sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -2421,6 +2654,7 @@ def wait_for_port_config_done(self, namespace):
             (key, op, fvp) = port_tbl.pop()
             if key in ["PortConfigDone", "PortInitDone"]:
                 break
+        sel.removeSelectable(port_tbl)
```

Comment:
in the newly added code it also subscribes to the same table, is possible to keep this subscription and reuse it?

Reply:
I have addressed this now.